.github/workflows/release_notes.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -ue
```

Comment:
I can see that we don't always use bash.

Reply:
I never use bash in release scripts as was formerly bit by doing things in docker with alpine. It is easier to never use it, as normal syntax is always around.

Reply:
basically you'll notice in scripts and also make, main dependencies will be sed, awk etc. even being really conservative things like windows translations bite.. we do what we can 😅